app/helpers/projects_helper.rb

```diff
@@ -43,4 +43,10 @@ def repository_web_link(project)
       ""
     end
   end
+
+  def docker_build_methods_help_text
+    Project::DOCKER_BUILD_METHODS.map do |method|
+      ["<b>#{method[:label]}</b>: #{method[:help_text]}"] unless method[:help_text].blank?
+    end.compact.join("<br/><br/>").html_safe
```

Comment:
XSS! :)

```
["<hacks>"].join("<br>").html_safe
=> "<hacks>"
```

Reply:
Since both the `label` and `help_text` aren't user supplied, is this still a concern?

Reply:
```Ruby
html = "".html_safe
foo.each do
```

Reply:
not really a concern here, but bad practice and might end up getting copy-pasted or learned by other devs ... so avoid :)

https://apidock.com/rails/ActionView/Helpers/OutputSafetyHelper/safe_join